### ISSUE TITLE
[Woo POS] M2: Improve accessibility: Labels

### ISSUE THREAD

**Issue by kidinov** [woocommerce/woocommerce-android#12305]
pfoUAQ-SL-p2

* The floating toolbar menu button doesn’t have an item label
* The cart’s product image has the same item description – we may want to put there a product name too
* Remove from the cart icon doesn’t have an item description
* The payment success screen checkmark says, “multiple times have the same description.”
* Add content description to the overlay that is shown when the floating toolbar menu is shown

**Comment by dangermattic** [comment]
🚫 Please add a type label (e.g. **type: enhancement**) and a feature label (e.g. **feature: stats**) to this issue.
